index.html

```diff
@@ -2641,61 +2644,64 @@ <h1>Servient Implementation</h1>
-                <a>Consumer</a> behavior (e.g., controlling a <a>Thing</a> or realizing mashups), and
-                <a>Intermediary</a> behavior (e.g., simply proxying a <a>Thing</a> or composing virtual entities).
-                The behavior implementation within a <a>Servient</a> defines
-                which <a>Things</a>, <a>Consumers</a>, and <a>Intermediaries</a> are hosted on this component.
+                The <em>behavior</em> defines the overall application logic of a <a>Thing</a>,
+                which has several aspects:
             </p>
             <p>
-                <a href="#architecture-implementation"></a> focuses on <a>Servients</a> implementing the optional <a>WoT Scripting API</a> building block,
+                It includes <em>autonomous behavior</em> of <a>Things</a> (e.g., sampling of sensors or control loops for actuators),
+                the <em>handlers</em> for <a>Interaction Affordances</a> (i.e., the concrete actions taken when an affordance is activated),
+                <em><a>Consumer</a> behavior</em> (e.g., controlling a <a>Thing</a> or realizing mashups), and
+                <em><a>Intermediary</a> behavior</em> (e.g., simply proxying a <a>Thing</a> or composing virtual entities).
+                The behavior implementation within a <a>Servient</a> defines which <a>Things</a>, <a>Consumers</a>, 
+                and <a>Intermediaries</a> are hosted on this component.
+            </p>
+            <p>
+                <a href="#architecture-implementation"></a> depicts <a>Servients</a> that are implementing the optional <a>WoT Scripting API</a> building block,
                 where portable application scripts written in JavaScript [[ECMAScript]] define the behavior.
                 They are executed by a scripting runtime system that is part of the <a>WoT Runtime</a>
                 (when providing the <a>WoT Scripting API</a> or any other script-based API).
                 They are portable, as they are written against the common <a>WoT Scripting API</a> definitions,
                 and hence can be executed by any <a>Servient</a> featuring this building block.
-                This makes it possible to migrate certain application logic between system components,
+                This makes it possible to shift application logic between system components,
                 for instance moving a <a>Consumer</a> from the cloud to an edge node to meet networking requirements,
-                or move an <a>Intermediary</a> to the cloud to fulfill growing resource demands.
-                The portability can also foster <a>Thing</a> customization by installing behavior following the app concept known from smartphones.
+                or to move an <a>Intermediary</a> to the cloud to fulfill growing resource demands.
+                Additional behavior can be 'installed' after deployment of a <a>Servient</a> through portable applications. 
```

Comment:
So need portable installer applications? :)

"Portable application scripts also allow to 'install' additional behavior after the deployment of a Servient."

?

Reply:
Your proposal also suggests that the applications provide the 'install' feature,
however this is a feature of the runtime.

Rewirded to:
Portable applications enable to 'install' additional behavior after the deployment of a Servient.